db/test/test_data.py

```diff
@@ -260,9 +263,20 @@ def test_add_get_model(self):
         get_id = db.data._get_model_id("modelname", "v1")
         self.assertEqual(modelid, get_id)
 
-
     def test_get_summary_data(self):
         pass
+    
+    def test_find_md5_duplicates(self):
+        
+        self.assertEqual(db.data.find_md5_duplicates(self.test_md5), None)
+        self.assertIsNone(db.data.find_md5_duplicates(self.test_md5))
+        db.data.submit_low_level_data(self.test_mbid, self.test_lowlevel_data)
+        self.assertEqual(str(db.data.find_md5_duplicates(self.test_md5)), self.test_mbid)
+        
+    def test_generate_datasetitem_uuidv4(self):
+        
+        self.assertTrue(db.data.generate_datasetitem_uuidv4())
```

Comment:
Oh, I thought it was related to check I was doing int the function not to the function itself.